web/ui/react-app/src/pages/agent/Agent.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FC } from 'react';
+
+export const AgentContent: FC = () => {
+  return (
+    <>
+      <h2>Prometheus Agent</h2>
+      <p>
+        This Prometheus instance runs in <strong>agent mode</strong>. In this mode, Prometheus is only used to scrape
+        discovered targets and forward them to remote write endpoints.
+      </p>
+      <p>The PromQL editor, graph console, recording rules and alerting page are not available in this mode.</p>
```

Comment:
The use of "PromQL editor" is a little confusing here because there is no dedicated page for the PromQL editor, it's included in the next page mentioned in the list.

Reply:
Should we add the TSDB status page here as well?